### DISCUSSION TITLE
Guess country

### DISCUSSION THREAD

**Discussion by SANKARAMDAS** [ParanoidUser/yolo#731] [Galaxy Trivia] [answered]
The Rainbow Mountain  -
Blue Forest -
Majuli island -
Strait Cataract -

**Comment by peterbaccaro** [comment, marked as answer]
1. The Rainbow Mountain
- **PERU**: The Rainbow Mountain, also known as Vinicunca or Montaña de Siete Colores in Spanish, is a mountain located in the Andes of Peru, South America.

2. Blue Forest
- **BELGIUM**: One of the most well-known Blue Forests is the Hallerbos, which is located in Belgium, about 20 km (12 miles) south of Brussels. However there are several forests that are referred to as the "Blue Forest" such as the Blue Forest in Spain, located in the province of León.

3. Majuli island
- **INDIA**: Majuli is a river island located in the Brahmaputra River, in the Indian state of Assam. It is considered to be the largest river island in the world, with an area of around 880 square kilometers.

4. Strait Cataract
- **ICELAND** and **GREENLAND**: The Denmark Strait Cataract is an international waterway, so it does not belong to any particular country.  The waterfall is located approximately halfway between Iceland and Greenland (an autonomous territory of Denmark).
